includes/class-newspack-popups-inserter.php

```diff
@@ -221,12 +221,21 @@ function ( $popup ) use ( $shortcoded_popups_ids ) {
 			$content = scaip_maybe_insert_shortcode( $content );
 		}
 
-		// Dynamic blocks might render an arbitrary amount of content, so the length of the content string is
-		// not an accurate representation of the content length.
-		$total_length = 0;
-		foreach ( parse_blocks( $content ) as $block ) {
-			$block_content = $block['innerHTML'];
-			$total_length += strlen( wp_strip_all_tags( $block_content ) );
+		// For certain types of blocks, their innerHTML is not a good representation of the length of their content.
+		// For example, slideshows may have an arbitrary amount of slide content, but only show one slide at a time.
+		// Don't insert prompts adjacent to these.
```

Comment:
Nit: this line in the comment is not accurate after the recent changes ;)

Reply:
Updated in f479c4d 😄